src/licensedcode/data/rules/proprietary-license_518.RULE

```diff
@@ -0,0 +1,4 @@
+Created as part of the Archimedes project (tools for
```

Comment:
`Created as part of the Archimedes project` may be enough for this rule with minimum_coverage: 100`